app/content/drill/[id]/description.js

```diff
@@ -17,6 +17,9 @@ import { useDrillInfo } from "~/hooks/useDrillInfo";
 
 export default function Description() {
   const drillId = useLocalSearchParams()["id"];
+  const assigned_time = useLocalSearchParams()["assigned_time"];
```

Comment:
Can you convert all assigned_time to assignedTime? We're a camelCase household.
Can you also change the putt_keys and putt_values in Utility to camelCase?